backend/internal/server/handlers.go

```diff
@@ -499,3 +499,69 @@ func GetSequencing(repo repository.StarrocksDAO) gin.HandlerFunc {
 		c.JSON(http.StatusOK, sequencing)
 	}
 }
+
+// MondoTermAutoComplete handles retrieving mondo terms by autocomplete
+// @Summary Get AutoCompleteTerm list of matching input string with highlighted
+// @Id mondoTermAutoComplete
+// @Description Retrieve AutoCompleteTerm list of mondo terms matching input string with highlighted
+// @Tags mondo
+// @Security bearerauth
+// @Param			message	body		types.AutoCompleteTermBody	true	"AutoCompleteTermBody"
+// @Accept json
+// @Produce json
+// @Success 200 {array} types.AutoCompleteTerm
+// @Failure 500 {object} map[string]string
+// @Router /mondo/autocomplete [post]
+func MondoTermAutoComplete(repo repository.StarrocksDAO) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		var (
+			body types.AutoCompleteTermBody
```

Comment:
In clin we use a queryParam for the input named `prefix` ofc it doesnt matter really but we could use a GET endpoint instead of POST this way, up to you.

Reply:
makes sense

Reply:
Done